apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx

```diff
@@ -196,6 +196,21 @@ export function DashboardsIndexPage() {
                 defaultMessage: 'Select dashboard checkbox',
                 description: 'dashboards table selection checkbox label',
               }),
+            cancelEditLabel: () =>
+              intl.formatMessage({
+                defaultMessage: 'Cancel table inline edit',
```

Comment:
Minor: I think the wording is better as `Cancel inline table edit`

Reply:
done